src/core/manager-serialize.c

```diff
@@ -550,6 +552,17 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                          * remains set until all serialized contents are handled. */
                         if (deserialize_varlink_sockets)
                                 (void) varlink_server_deserialize_one(m->varlink_server, val, fds);
+                } else if ((val = startswith(l, "dump-ratelimit="))) {
+                        usec_t begin;
+                        unsigned num;
+
+                        if (sscanf(val, USEC_FMT " %u", &begin, &num) != 2)
+                                log_notice("Failed to parse dump ratelimit, ignoring: %s", val);
+                        else {
+                                m->dump_ratelimit.begin = begin;
+                                m->dump_ratelimit.num = num;
+                        }
```

Comment:
hmm, probably would make sense to also save/restore interval/burst, in case this is changed one day. maybe if interval/burst don't match when parsing simply flush the counter out.